flow_control/flow.py

```diff
@@ -347,6 +386,14 @@ def _calculate_total(first_score, second_score):
         scores = map(scores_client.get, usages_keys)
         scores = filter(None, scores)
 
+        problems_to_answer = [score.total for score in scores]
```

Comment:
Yes,  `None` scores on a problem will be map to Zero when the selected operator does not check null conditions.